trie-db/src/triedb.rs

```diff
@@ -15,10 +15,10 @@
 #[cfg(feature = "std")]
 use crate::nibble::NibbleVec;
```

Comment:
```suggestion
use crate::{nibble::NibbleVec, node::Node};
```
Maybe we can group the stuff to be included with `std` under a single directive, thus also the `crate::rstd::{fmt, vec::Vec}` below